io.asgardio.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+consumerKey=KE4OYeY_gfYwzQbJa9tGhj1hZJMa
+consumerSecret=_ebDU3prFV99JYgtbnknB0z0dXoa
```

Comment:
Let remove values like these

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/7/commits/bf4f7fa7367a16bd78c9cbabc954df5fb7451e18